app/src/main/res/raw/contributors.txt

```diff
@@ -198,6 +200,7 @@ OPADILOP
 DivideEtImpera
 Nicolas
 Nosnahc
+osamaqw
```

Comment:
I think this is the person who renamed Search to Osama in Weblate.

Best case we start figuring out when there is 0 code of a person left, but that's hard. A minimum commit count is also possible but that would exclude more valid contributions.

Probably should just extend the exclude list. Or limit it to showing a certain amount of people and use "and more".

Or maybe just ignore this false positive and allow them on because they probably tried to do good but didn't understand Weblate 😅

Reply:
Maybe give them a chance to actually translate until the next release? Remove them later if they never contribute anything. idk.